docs/api/rest.md

```diff
@@ -208,6 +208,8 @@ You can pass the following optional GET parameters:
 - `minimal_response` to only return `last_changed` and `state` for states other than the first and last state (much faster).
 - `no_attributes` to skip returning attributes from the database (much faster).
 - `significant_changes_only` to only return significant state changes.
+	
+Note: They must be placed in this order.
```

Comment:
This is not true. 

https://github.com/home-assistant/core/blob/dev/homeassistant/components/history/__init__.py#L112

Reply:
Ok, I think I found out what happened. 

I had been testing with two URLs:

This one returned a 200 code.
```
https://<my_ip_here>/api/history/period/2023-03-16T21:44:00+00:00?filter_entity_id=sensor.program_messages?end_time=2023-03-17T20:44:19+00:00
```

This one returned a 400 code.
```
https://<my_ip_here>/api/history/period/2023-03-16T21:48:00+00:00?end_time=2023-03-17T20:48:10+00:00?filter_entity_id=sensor.program_messages
```

Because one returned 200, I had assumed that meant my request was ok, and I wasn't able to parse the data because Javascript was being weird or something, and that Home Assistant just required the parameters in a certain order.

I now know the mistake I made, I had assumed that you were supposed to separate the URL parameters with a ?, but it's actually an &, which I only figured out when I put the URL in postman and the rows didn't make sense.

So I would now like to propose a different change to make it more difficult for somebody to make the same mistake I did, by adding an example that uses three parameters.

Reply:
I don't think we need to add this. This is truly the bare basics of URLs we are trying to solve now, which is out of scope here.

Nevertheless, thanks for being willing to contribute. 👍 

../Frenck